holoviews/plotting/bokeh/renderer.py

```diff
@@ -56,23 +59,9 @@ def __call__(self, obj, fmt=None):
             plotobjects = [h for handles in plot.traverse(lambda x: x.current_handles)
                            for h in handles]
             data = dict(data=[])
-            ids = []
             if not old_bokeh:
                 data['root'] = plot.state._id
-            json_data = []
-            for plotobj in plotobjects:
-                if plotobj.ref['id'] in ids:
-                    continue
-                else:
-                    ids.append(plotobj.ref['id'])
-                if old_bokeh:
-                    json = plotobj.vm_serialize(changed_only=True)
-                else:
-                    json = plotobj.to_json(False)
```

Comment:
If you find this pattern happening often, I would consider using a utility what does the version check and smooths over the differences between bokeh versions. I'll let you decide if this this worth it - if you only need it in this one instance, it probably doesn't need to be made into a utility right now.